content/v2.0/reference/cli/influx/delete/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: influx delete – Delete data from InfluxDB
+description: The 'influx delete' command deletes points from InfluxDB.
+menu:
+  v2_0_ref:
+    name: influx delete
+    parent: influx
+weight: 101
+v2.0/tags: [delete]
+---
+
+The `influx delete` command deletes points from InfluxDB.
+
+## Usage
+```
+influx delete [flags]
+```
+
+{{% warn %}}
+Running `influx delete` without the `-p` or `--predicate` flag will delete all data
+with timestamps between the specified `--start` and `--stop` times.
```

Comment:
Just curious: Are there default start/stop times? In other words, what happens if I just run `influx delete`?

Reply:
You have to provide an org, token, bucket, start, and stop. It will return an error if you don't